logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
+      $debugging_event = $event;
+      unset($debugging_event['type']);
```

Comment:
Such a hook is a good idea for individual extra debugging requirement but that can already be done if a custom module implement hook_watchdog(). But this module should provide a useful default so that people can start using it without having to write a custom module.

So, our difference is whether the debugging information should be included once per request or any number of times per request. Remember, the arrays included can be really verbose and I fear it gets difficult to debug if you have all that information - which is always the same - included maybe dozens of times.

However, if you still think it should be done that way, why don't we leave that to the site owner and allow them to decide themselves by providing an option in the settings?